package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
yeah it will work, but what if it really doesn't support istanbul 5.X? If it is only a devDependency then this lib will break. It should have a dependency on the lib version it actually supports.